tests/integration/test_distributed_system_flush_settings_query/test.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+node1 = cluster.add_instance("node1", main_configs=["configs/remote_servers.xml"])
+node2 = cluster.add_instance("node2", main_configs=["configs/remote_servers.xml"])
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+
+        for node in (node1, node2):
+            node.query(
+                """CREATE TABLE local_table(id UInt32, val String) ENGINE = MergeTree PARTITION BY id ORDER BY id;"""
+            )
+
+        node1.query(
+            """CREATE TABLE distributed_table(id UInt32, val String) ENGINE = Distributed(test_cluster, default, local_table, id);"""
+        )
+
+        yield cluster
+
+    finally:
+        cluster.shutdown()
+
+
+def test_start_and_stop_replica_send(started_cluster):
+    node1.query("SYSTEM STOP DISTRIBUTED SENDS distributed_table;")
+    node1.query("SET max_partitions_per_insert_block = 1;")
+
+    node1.query("INSERT INTO distributed_table VALUES (0, 'node1'),(1, 'node2')")
+
+    # Write only to this node when stop distributed sends
+    assert node1.query("SELECT COUNT() FROM distributed_table").rstrip() == "1"
+
+    node1.query("SET max_partitions_per_insert_block = 0;")
+    node1.query("SYSTEM START DISTRIBUTED SENDS distributed_table;")
+    node1.query("SYSTEM FLUSH SETTINGS DISTRIBUTED distributed_table;")
```

Comment:
Actually I think it should be `SYSTEM FLUSH DISTRIBUTED SENDS db.table SETTINGS ...`
In other words simply add `SETTINGS` clause support for `SYSTEM FLUSH DISTRIBUTED SENDS`

Reply:
> Actually I think it should be `SYSTEM FLUSH DISTRIBUTED SENDS db.table SETTINGS ...` In other words simply add `SETTINGS` clause support for `SYSTEM FLUSH DISTRIBUTED SENDS`

OK